src/dos/dos_misc.cpp

```diff
@@ -66,6 +66,9 @@ static Bitu INT2A_Handler(void) {
 static bool DOS_MultiplexFunctions(void) {
 	switch (reg_ax) {
 	case 0x1000:
+		if (!emulate_file_locking) {
```

Comment:
Yeah, so as @interloper98 suggested, can you please introduce a `DOS_IsFileLocking()` function instead? I'm really not a fan of using globals like this.

Reply:
Changed